### PR TITLE
Type of the cluster is aligned vertically in gluster detail page

### PR DESCRIPTION
**Pull request by Prabu-raj**
Aligned vertically from bottom to top

### CODE REVIEW

File: app/views/hosts.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
           <c3chart bindto-id="donut-cpu-chart-{{$index}}">
               <chart-column column-id="Free" column-values="{{100-host.cpu_average}}" column-color="#CCCCCC" column-type="donut" />
               <chart-column column-id="Using" column-values="{{host.cpu_average}}" column-color="{{hosts.getHostDonutColor(host.cpu_average)}}" column-type="donut" />
-              <chart-donut title="{{host.cpu_average}}%" width="10" />
+              <chart-donut order="null" title="{{host.cpu_average}}%" width="10" />
```
Comment (kmkanagaraj) [inline app/views/hosts.html]: Is this change required?

If not, can revert this and amend?

File: app/views/hosts.html [hunk 2]
```diff
@@ -43,7 +43,7 @@
           <c3chart bindto-id="donut-memory-chart-{{$index}}">
               <chart-column column-id="Free" column-values="{{100-host.memory_average}}" column-color="#CCCCCC" column-type="donut" />
               <chart-column column-id="Using" column-values="{{host.memory_average}}" column-color="{{hosts.getHostDonutColor(host.memory_average)}}" column-type="donut" />
-              <chart-donut title="{{host.memory_average}}%" width="10" />
+              <chart-donut order="null" title="{{host.memory_average}}%" width="10" />
```
Comment (kmkanagaraj) [inline app/views/hosts.html]: same here

File: app/views/hosts.html (hunk 1 above)
Comment (Prabu-raj) [inline app/views/hosts.html]: Not required. Forgot to change. I will ammend now.
